.github/workflows/docker.yml

```diff
@@ -0,0 +1,45 @@
+name: Docker
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+#      - name: Lint Dockerfile
```

Comment:
i commented this out as hadolint suggests that all the versions of apk packages are pinned. 
Options are:
* DIsable hadolint (did it)
* Pin Versions
* Put a hadolintignore in